jobs/fetch_resources.go

```diff
@@ -1,5 +1,12 @@
 package jobs
 
-func (h *handler) FetchResources() error {
-	return h.providerService.FetchResources()
+import (
+	"context"
+
+	"github.com/odpf/guardian/pkg/audit"
+)
+
+func (h *handler) FetchResources(ctx context.Context) error {
+	ctx = audit.WithActor(ctx, "system")
```

Comment:
saw this in several places, should we define a new function called `WithSystemActor` ?

Reply:
the current one is still simple enough I would say, it is also just used in two different places

Reply:
Hmm true but IMO defining it in one place and reuse it in other places would make it more consistent (in case there is a typo) and easier to refactor later on.